config/locales/candidate_interface/guidance.yml

```
@@ -0,0 +1,24 @@
+en:
+  candidate_interface:
+    guidance:
+      index:
+        heading: The application process for courses starting in September %{this_cycle}
+        you_can_start_applying: For courses starting in September %{this_cycle}, you can start applying a year before they start in October %{last_cycle}.
+        duration: Most teacher training courses are one year and start in September when the academic year in England starts. Part time courses are usually 2 years.
+        find_courses_html: "Start %{link} you might want to apply to."
+        start_applying: |
+          Start applying to courses. You can apply to %{max_courses} courses, they can fill up quickly so you should apply as soon as you can.
+
+          Providers can close courses to new applications when they fill up.
+
+          You can withdraw an application at any time and apply to a different course if you want to.
+
+          If training providers do not respond to an application within 30 working days, you will be able to apply to another course while you wait for a response. We will tell you if the provider has not responded in time.
```

There is a later ticket https://trello.com/c/2NXJTv53 that has a change to this text. Can we incorporate it here (avoiding merge conflicts)
```suggestion
          If training providers do not respond to an application within 30 working days, you will be able to apply to another course while you wait for a response. We will tell you if the provider has not responded in time. Around the Easter and Christmas breaks, providers can take longer to respond to applications.
```